### DISCUSSION TITLE
Documentation Bug: Links in Config Comments are Dead

### DISCUSSION THREAD

**Discussion by fkellner** [spatie/laravel-markdown#4] [Q&A] [answered]
Dear Spatie Team,

after publishing your markdown config, I found links like [this](https://github.com/spatie/laravel-markdown#customizing-the-rendering-process) everywhere in the comments. As you can see when clicking on them, they are dead. 

Please update them to point to the actual documentation.

Best Regards

**Comment by fkellner** [comment, marked as answer]
https://github.com/spatie/laravel-markdown/pull/5

**Reply by freekmurze** [reply]
Thanks!
